lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/client/YarnSubmissionHelper.java

```diff
@@ -241,6 +242,29 @@ public YarnSubmissionHelper setConfigurationFilePaths(final List<String> configu
   }
 
   /**
+   * Sets environment variable map.
+   * @param map
+   * @return
+   */
+  public YarnSubmissionHelper setJobSubmissionEnvMap(final Map<String, String> map) {
+    for (final Map.Entry<String, String> entry : map.entrySet()) {
+      envMap.put(entry.getKey(), entry.getValue());
+    }
+    return this;
+  }
+
+  /**
+   * Adds a job submission environment variable.
+   * @param key
+   * @param value
+   * @return
+   */
+  public YarnSubmissionHelper addJobSubmissionEnvVariable(final String key, final String value) {
```

Comment:
rename to `setJobSubmissionEnvironmentVariable`

Reply:
Done